File: CoreJava/iemotionable-app/src/com/techlab/human/Men.java
package com.techlab.human;

import com.techlab.interfaces.IEmotionable;
import com.techlab.interfaces.ISocializable;

public class Men implements IEmotionable, ISocializable {

	@Override
	public void wish() {
		System.out.println("Men is Wishing...");
	}

	@Override
	public void depart() {
		System.out.println("Men is Departing...");
	}

	@Override
	public void cry() {
		System.out.println("Men is Crying...");

	}

	@Override
	public void laugh() {
		System.out.println("Men is Laughing...");
	}

}

File: C#Basics/Interface-App/Human-Behaviour-App/Men.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Human_Behaviour_App
{
    class Men : IEmotionable, ISocializable
    {
        void ISocializable.wish()
        {
            Console.WriteLine("Men is Wishing...");
        }


        void ISocializable.depart()
        {
            Console.WriteLine("Men is Departing...");
        }


        void IEmotionable.cry()
        {
            Console.WriteLine("Men is Crying...");

        }


        void IEmotionable.laugh()
        {
            Console.WriteLine("Men is Laughing...");
        }
    }
}
